### PR TITLE
new edit test case page

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: edit-test-case.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- Select a different locator with your mouse in the snapshot
+
+<Frame caption="Changing locators with the virtual locator picker, 7/2024">
+  <img
+    src="/images/editing/change-locator.gif"
+    alt="changing locators with virtual locator picker"
+  />
+</Frame>
+
+### Change locator type
```
Comment (sker65) [inline edit-test-case.mdx:27]: I guess you not change the locator type but the interaction type

Comment (ma-zah) [inline edit-test-case.mdx:27]:
```suggestion
### Change interaction or assertion type
```

File: edit-test-case.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+<Frame caption="Drag & drop a new step, delete a step, 7/2024">
+  <img
+    src="/images/editing/drag-n-drop.gif"
+    alt="drag and drop a test step and delete it"
+  />
+</Frame>
+
+### Drag & drop, duplicate and copy paste steps
+
+Octomind test steps can be flexibly manipulated. You can **drag and drop** steps to different places in the sequence, duplicate them by clicking on the `duplicate` icon in the upper right corner or simply `select` a step and **copy & paste** it where you need it. `Ctrl+C` and `Ctrl+V` works also for copy pasting steps from other test cases.
```
Comment (sker65) [inline edit-test-case.mdx:65]: maybe add mac keyboard shortcuts cmd-c / cmd-v ....

File: edit-test-case.mdx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  <img src="/images/editing/duplicate-step.png" alt="duplicating test steps" />
+</Frame>
+
+## Edit prompt and restart AI auto-generation
+
+You can edit the prompt and have the AI agent `regenerate` test steps based on the new prompt. We will ask you to select **the last step** in the sequence you want to keep. The AI agent will regenerate test steps from this step onwards.
+
+Or you have simply hit the `regenerate` button to rediscover the steps based on the same prompt.
+
+<Frame caption="Drag & drop a new step, delete a step, 7/2024">
```
Comment (sker65) [inline edit-test-case.mdx:77]: maybe wrong caption? copy / paste from above?

File: edit-test-case.mdx (hunk 2 above)
Comment (ma-zah) [inline edit-test-case.mdx]:
```suggestion
Octomind test steps can be flexibly manipulated. You can **drag and drop** steps to different places in the sequence, duplicate them by clicking on the `duplicate` icon in the upper right corner or simply `select` a step and **copy & paste** it where you need it. `Ctrl+C` and `Ctrl+V` / `cmd+c` and `cmd+v` works also for copy pasting steps from other test cases.
```

Comment (ma-zah) [reply edit-test-case.mdx]: ok

Comment (ma-zah) [inline edit-test-case.mdx]:
```suggestion
Octomind test steps can be flexibly manipulated. You can **drag and drop** steps to different places in the sequence, duplicate them by clicking on the `duplicate` icon in the upper right corner or simply `select` a step and **copy & paste** it where you need it. `Ctrl+C` and `Ctrl+V` or `cmd+c` `cmd_v` works also for copy pasting steps from other test cases.
```

File: edit-test-case.mdx (hunk 3 above)
Comment (ma-zah) [inline edit-test-case.mdx]:
```suggestion
<Frame caption="Edit prompt and regenerate the test step, 7/2024">
```

**Review by ma-zah — COMMENTED** [review]
ok

**Review by Germandrummer92 — APPROVED** [review]

**Pull request merged**